src/modules/bulk-minter/components/Loading.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from "react";
+
+interface LoadingProps {
+    className?: string
+    w?: string
+    h?: string
+  }
+  
+const Loading: FunctionComponent<LoadingProps> = ({
+    className,
+    w = 5,
+    h = 5,
+  }) => {
```

Comment:
We are starting to use this pattern everywhere

```suggestion
export function Loading(props: LoadingProps): JSX.Element {
```

and then access your props without deconstruction, i.e. `props.className`